src/history/createStateHistory.ts

```diff
@@ -87,31 +73,42 @@ const createStateHistory: StateHistoryFactory = compose.mixin(createEvented, {
 			return privateStateMap.get(this).current;
 		},
 
+		prefix(this: StateHistory, path: string) {
+			const { base } = privateStateMap.get(this);
+			const baseEndsWithSlash = /\/$/.test(base);
```

Comment:
Nope, this is the only occurrence of that regular expressions. The one on the next line (`/^\//`) is used twice, but regular expressions are cached anyway in the VMs so no need to assign them to a variable if that's what you were suggesting.